bucket/tfsec.json

```
@@ -0,0 +1,26 @@
+{
+    "version": "0.58.9",
+    "description": "tfsec uses static analysis of your terraform templates to spot potential security issues.",
+    "homepage": "https://github.com/aquasecurity/tfsec",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/aquasecurity/tfsec/releases/download/v0.58.9/tfsec-windows-amd64.exe",
+            "hash": "b625805dd606c20a0fc75e3beeac661e98725d1cfb61598952b4cec3fd3b2039"
+        }
+    },
+    "bin": "tfsec.exe",
+    "checkver": {
+        "github": "https://github.com/aquasecurity/tfsec"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/aquasecurity/tfsec/releases/download/v$version/tfsec-windows-amd64.exe"
```

The executable needs re-mapping.

```suggestion
                "url": "https://github.com/aquasecurity/tfsec/releases/download/v$version/tfsec-windows-amd64.exe#/tfsec.exe"
```
```suggestion
                "url": "https://github.com/aquasecurity/tfsec/releases/download/v$version/tfsec-windows-amd64.exe"
```